tests/utils/snapPoints.js

```diff
@@ -39,7 +39,32 @@ const isAtPercentageSnapPoint = async (browser, cardSelector, expectedSnapPointP
     return true
 }
 
+const isAtSnapPoint = async (browser, cardSelector, snapPointValue)=>{
+    let check;
+
+    if (snapPointValue <= 1){
+        check = checkPercentageSnapPoint;
+    } else {
+        check = checkAbsoluteSnapPoint;
+    }
+
+    let isAtSnapPoint = false;
+
+    await browser.waitUntil(async ()=>{
+        isAtSnapPoint = await check(browser, cardSelector, snapPointValue)
+
+        return isAtSnapPoint;
+    }, {
+        timeout: 5000,
+        interval: 500,
+        timeoutMsg: `Card is not at expected snapPoint: ${snapPointValue}`
+    });
+
+    return isAtSnapPoint;
+}
+
 module.exports = {
-    isAtPercentageSnapPoint,
-    isAtAbsoluteSnapPoint
+    isAtSnapPoint,
+    checkAbsoluteSnapPoint,
+    checkPercentageSnapPoint
```

Comment:
Is there a reason to expose this methods? Should we only expose the one choosing the strategy instead?